lib/child_process.js

```diff
@@ -78,7 +90,7 @@ function fork(modulePath /* , args, options */) {
   }
 
   if (pos < arguments.length &&
-      (arguments[pos] === undefined || arguments[pos] === null)) {
+    (arguments[pos] === undefined || arguments[pos] === null)) {
```

Comment:
This change seems unrelated.

Reply:
Thanks, these unrelated changes were caused by running "make js-lint-fix", now i revert the code